examples/sample_01/terraform.tfvars

```diff
@@ -0,0 +1,2 @@
+project                  = "koller-dani-sandbox"
+credentials_file         = "C:/Users/kollerdani/Desktop/koller-dani-sandbox-10e37a32eb02.json"
```

Comment:
in general, you should never put anything into the code repository that only works in your local environment.